include/full_coverage_path_planner/common.h

```diff
@@ -38,6 +38,8 @@ typedef struct
 }
 gridNode_t;
 
+enum Direction {NORTH, EAST, SOUTH, WEST};
```

Comment:
Created Enumerator to store ants current direction.

This is so ants can easily remember the direction they were moving in the previous iteration, all the program needs to do is repeat this direction.